packages/@okta/i18n/src/properties/login_ok_PL.properties

```diff
@@ -668,7 +669,7 @@ oie.password.showConfirmPassword = 》Šĥöŵ ŕé‐éñţéŕéð þåššŵ
 oie.password.hidePassword = 》Ĥîðé þåššŵöŕð ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.password.hideConfirmPassword = 》Ĥîðé ŕé‐éñţéŕéð þåššŵöŕð 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.password.reset.verification = 》Ṽéŕîƒý ŵîţĥ öñé öƒ ţĥé ƒöļļöŵîñĝ šéçûŕîţý ɱéţĥöðš ţö ŕéšéţ ýöûŕ þåššŵöŕð· 홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
-idx.password.expiring.message = 》Ŵĥéñ ýöûŕ þåššŵöŕð éẋþîŕéš ýöû ŵîļļ ƀé ļöçķéð öûţ öƒ ýöûŕ Öķţå åççöûñţ· 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+idx.password.expiring.message = 》Ŵĥéñ ýöûŕ þåššŵöŕð éẋþîŕéš، ýöû ŵîļļ ĥåṽé ţö çĥåñĝé ýöûŕ þåššŵöŕð ƀéƒöŕé ýöû çåñ ļöĝîñ ţö ýöûŕ ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ {0} åççöûñţ· ฐโ《
```

Comment:
fyi @jaredperreault-okta this got picked up when i ran pseudo-loc. no action needed unless this shouldn't change